src/main/scala/introcourse/level01/IntroExercises.scala

```diff
@@ -35,7 +35,10 @@ object IntroExercises {
     * = 9
     *
     **/
-  def add5(x: Int): Int = ???
+  def add5(x: Int): Int = {
+    val f = ???
```

Comment:
maybe a type annotation on `f` so that people assign it to the right thing?  Otherwise they could just do:
```
val f = add(5)(x)
f
```

Reply:
Great point! Change made.

I don't understand the CI failure, something to do with oraclejdk :/

Reply:
I think the CI has been fixed, might just need a rebase.